### ISSUE TITLE
DatePlot project doesn't build because it's using the Swift package

### ISSUE THREAD

**Issue by DivineDominion** [core-plot/core-plot#468]
I noticed that Carthage fails in Xcode 13.4.1 and 14.0:

> Packages are not supported when using legacy build locations, but the current project has them enabled.

The `examples/DatePlot/DatePlot.xcodeproj` is being used by Carthage -- which is probably also not correct, but that's another issue.

The `DatePlot.xcodeproj` also provides the same error in Xcode directly.

All the other example projects work fine since they don't use the Swift package output.

## Full error message

```
$ carthage build --no-skip-current
*** xcodebuild output can be found in /var/folders/62/8k21681d08z9lhq8h433z3rh0000gp/T/carthage-xcodebuild.orx6Mt.log
A shell task (/usr/bin/xcrun xcodebuild -project /Users/ctm/Downloads/core-plot/examples/DatePlot/DatePlot.xcodeproj CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES -list) failed with exit code 74:
2022-10-14 08:51:29.997 xcodebuild[4301:1524260] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-10-14 08:51:29.997 xcodebuild[4301:1524260] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
xcodebuild: error: Could not resolve package dependencies:
  Packages are not supported when using legacy build locations, but the current project has them enabled.
```

**Comment by DivineDominion** [comment]
The folks at Flutter had similar problems with CI:
https://github.com/flutter/flutter/issues/76868#issuecomment-815027547

Some answers on the web say it's an Xcode bug. 🤷 

I've messed with the build settings and removed the `xcuserdata/` subfolders to no avail.

I've changed the Xcode-generated `DatePlot.xcodeproj/project.xcworkspace/xcuserdata//MYUSERNAME.xcuserdatad/WorkspaceSettings.xcsettings` file to use the value the Flutter folks recommend, changing

	<key>BuildLocationStyle</key>
	<string>UseTargetSettings</string>

to

	<key>BuildLocationStyle</key>
	<string>UseAppPreferences</string>

Since the Xcode prefs in my case use the new build system, this seems to win.

That's not a real fix, though.
